com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs

```diff
@@ -513,7 +513,7 @@ public static async Task<string> ReadAllTextAsync(string path)
         {
 #if UNITY_ANDROID && !UNITY_EDITOR
             // On Android, use a custom helper to read the file synchronously
-            return await Task.FromResult(AndroidFileIOHelper.ReadAllText(path));
+            return await AndroidFileIOHelper.ReadAllText(path);
```

Comment:
Is this actually running synchronously? This looks like a normal async Task thing to me, maybe comment needs changing or we need to make sure to do this synchronously?

Reply:
Comment needs changing - I've updated it to reflect that.